docs/_templates/components_requirements.tmpl

```diff
@@ -1,39 +1,83 @@
 .. -*- mode: rst -*-
 
-Inmanta core and extensions versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The orchestrator is constituted of several components (python packages and npm packages).
-The python packages are of two types: one core package and several extension packages.
 
-{% if data['component_requirements']%}
 
-The following table shows the ranges of the Inmanta Python components, this product version is compatible with.
-The lower bound of these version constraints will remain the same for the whole lifetime of this major version. Upper bounds may be added when a new major version is released.
+{% if data['module_compatibility_ranges']%}
 
-This table is only meant as a reference for developers of Inmanta extensions and is meant to be used as input to a
-dependency resolver e.g. pip to ensure the resulting versions are compatible.
-This information is not relevant when installing the orchestrator via RPM as is recommended
-in the :ref:`install-server` section.
 
-.. list-table:: Inmanta core and extensions versions
-   :header-rows: 1
+Module compatibility ranges
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   * - .. centered:: Component
-     - .. centered:: Required version
 
-   {% for key, value in data["component_requirements"] | dictsort %}
-   * - .. centered:: {{ key }}
-     - .. centered:: {{ value }}
-   {% endfor %}
+When an Inmanta module integrates tightly with the Inmanta Service Orchestrator, it's important
+to use a version of that module that is compatible with the orchestrator. This can be achieved by either:
 
-.. note::
+- Using a specific index (or git repo) per ISO version and making sure it contains the compatible versions of the modules.
+- Using a generic index (or git repo) possibly containing multiple versions for a given module and adding a
```

Comment:
One pip index can host multiple modules, while there is only one git repo per module (V1). So repos has to be plural.
```suggestion
- Using a specific pip index (or git repos) per ISO version and making sure it only contains the compatible versions of the modules.
- Using a generic pip index (or git repos) containing module versions for different ISO versions and adding a
```